tests/bulk_create/models.py

```diff
@@ -64,9 +64,9 @@ class State(models.Model):
 
 
 class TwoFields(models.Model):
-    f1 = models.IntegerField(unique=True)
-    f2 = models.IntegerField(unique=True)
-    name = models.CharField(max_length=15, null=True)
+    f1 = models.IntegerField(unique=True, db_column="F1")
+    f2 = models.IntegerField(unique=True, db_column="F2")
+    name = models.CharField(max_length=15, null=True, db_column="Name")
```

Comment:
We should add a separate models and tests.

Reply:
Also, `fields` are not affected so you don't need to add `db_column` to the `name`.

Reply:
> We should add a separate models and tests.

But why? I think they will be similar to the existing ones 🤔

Reply:
> > We should add a separate models and tests.
> 
> But why? I think they will be similar to the existing ones thinking

Because now they don't cover previously covered cases, e.g. quoting identifiers on Oracle.

Reply:
okay, i'll add a new model and testcase.

Reply:
I would add:
```python
class MixedCaseFields(models.Model):
    rank = models.IntegerField(unique=True, db_column="rAnK")
    name = models.CharField(max_length=15, null=True, db_column="NaMe")
```

```python
    @skipUnlessDBFeature(
        "supports_update_conflicts", "supports_update_conflicts_with_target"
    )
    def test_update_conflicts_unique_fields_mixed_case(self):
        MixedCaseFields.objects.bulk_create(
            [
                MixedCaseFields(rank=1, name="a"),
                MixedCaseFields(rank=2, name="b"),
            ]
        )
        self.assertEqual(MixedCaseFields.objects.count(), 2)

        conflicting_objects = [
            MixedCaseFields(rank=1, name="c"),
            MixedCaseFields(rank=2, name="d"),
        ]
        MixedCaseFields.objects.bulk_create(
            conflicting_objects,
            update_conflicts=True,
            unique_fields=["rank"],
            update_fields=["name"],
        )
        self.assertEqual(MixedCaseFields.objects.count(), 2)
        self.assertCountEqual(
            MixedCaseFields.objects.values("rank", "name"),
            [
                {"rank": 1, "name": "c"},
                {"rank": 2, "name": "d"},
            ],
        )
```

Reply:
you did all the work 😿

Reply:
hahaha, I pushed edits. 👍